### ISSUE TITLE
Error Code : N-02-HF2JU

### ISSUE THREAD

**Issue by Dhruv-1608** [Tech-FZ/EmuGUI#95]
While installing a new machine for the Kali Linux, I am facing this error. I am using the latest version of the Kali Linux QEMU Virtual Machine version, and I am using the Latest EmuGUI as well. Any sort of help would be really appreciated. 

<img width="338" alt="image" src="https://github.com/user-attachments/assets/8fbe099c-51d2-41c4-b2f7-57e927bc7c5b" />

**Comment by Tech-FZ** [comment]
You are using a deprecated (in EmuGUI: depreciated) feature ehuch you should turn off. However, it should not prevent you from starting the VM yet.

**Comment by Dhruv-1608** [comment]
> You are using a deprecated (in EmuGUI: depreciated) feature which you should turn off. However, it should not prevent you from starting the VM yet.

Well yes the VM did started but it wasnt really working. I was able to get it started and stuff, but then was facing this issue for the particular Linux Machine
<img width="320" alt="image" src="https://github.com/user-attachments/assets/c36e24b0-328c-423c-a157-cd016d39897d" />
I am trying to install Kali Linux VM onto my windows 11 laptop and I am facing the following error. Any sort of help would be really appreciated.

**Comment by Tech-FZ** [comment]
Did you use the RAW file format with the `.img` file ending?

**Comment by Dhruv-1608** [comment]
No, well I installed it from the official Kail Linux Website for QEMU. The file extension was `.qcow2`. After that it wasnt working, I did also tried with the `.iso` file as well just to check which wasnt working either. The screenshot is of the `.qcow2` file.

**Comment by Tech-FZ** [comment]
Did you use the x86_64 or the ARM64 silicon version?

**Comment by Dhruv-1608** [comment]
x86_64 Version. If possible, then could you provide some tutorial or a way by which I can install it.

**Comment by Tech-FZ** [comment]
I will try to investigate your issue. In the meantime, may I know your host computer specifications, as well as which version of EmuGUI and QEMU you were running? This can help me to cope for any potential limitations.

**Comment by Tech-FZ** [comment]
I have experienced issues with WHPX. Please try TCG instead, bare with the installation, no matter how long it takes, and close the issue of that helped.
